web_src/js/index.js

```diff
@@ -6,6 +6,8 @@ import './publicPath.js';
 import './gitGraphLoader.js';
 import './semanticDropdown.js';
 
+import ActivityTopAuthors from './components/ActivityTopAuthors.vue';
```

Comment:
Outside this PR, we should maybe start doing code-splitting and lazy load components that are not always displayed/used now that we "build" all the frontend.